app/controllers/mixins/actions/vm_actions/ownership.rb

```diff
@@ -213,7 +213,11 @@ def ownership_handle_reset_button
 
         def filter_ownership_items(klass, ownership_ids)
           records = find_records_with_rbac(klass.order(:name), ownership_ids)
-          records.with_ownership if klass.respond_to?(:with_ownership)
+          if klass.respond_to?(:with_ownership)
+            records.with_ownership if klass.respond_to?(:with_ownership)
```

Comment:
@mzazrivec I guess it there is condition accidentally duplicated.

maybe together can be
```ruby
records.try(:with_ownership) || records
```

Reply:
@lpichler fixed, thanks.